pages/common/svn-changelist.md

```diff
@@ -0,0 +1,24 @@
+# svn changelist
+
+> Associate a changelist with a set of files.
+> More information: <https://subversion.apache.org>.
+
+- Add files to a [new] changelist:
+
+`svn changelist {{changelist_name}} [{{FILE_PATH}}]`
```

Comment:
By `[FILE_PATH]` do you mean `FILE_PATH` is optional or an array of `FILE_PATHS`?

- If you mean it's optional, I think it's better to not show that capability and just assume the user wants to add 1 or more files.
- If you mean it can be an array, then my suggestion below clarifies that and makes it more like our general style for arrays on the command line:

```suggestion
`svn changelist {{changelist_name}} {{path/to/file1}} {{path/to/file2}}`
```